include/parflow/pfdata.hpp

```diff
@@ -206,201 +390,165 @@ class PFData {
 	 * @return int
 	 */
     int getNY() const;
-	/** @param getN[X,Y,Z] [NZ] describe the dimensions of the
+
+	/** @param getN[Z,Y,X] [NX] describe the dimensions of the
 	 * computational domain. This function is useful either when reading an
 	 * existing file or when confirming the configuration of a file that is
 	 * being created where the computational grid has already been set.
 	 * @return int
 	 */
-    int getNZ() const;
-
-    /**
-     * setN[X,Y,Z]
-     * @param double N[X,Y,Z]
-     * [NX,NY,NZ] describes the dimensions of the computational grid.
-     * This function is useful when creating a new pfb file. It is important to note that you can call this
-     * function on an existing file, but it will invalidate the file and break all subsequent uses of the class
-     * unless you call load_file() again to reset the value back to the one used in the file.
-     */
+    int getNX() const;
 
-	/** @param double [NX,NY,NZ] [NX] describes the dimensions of the
+	/** @param double [NZ,NY,NX] [NZ] describes the dimensions of the
 	 * computational grid. This function is useful when creating a new pfb file.
 	 * It is important to note that you can call this function on an existing
 	 * file, but it will invalidate the file and break all subsequent uses of
 	 * the class unless you call load_file() again to reset the value back to
 	 * the one used in the file.
 	 */
-    void setNX(int NX);
-	/** @param double [NX,NY,NZ] [NY] describes the dimensions of the
+    void setNZ(int NZ);
+
+	/** @param double [NZ,NY,NX] [NY] describes the dimensions of the
 	 * computational grid. This function is useful when creating a new pfb file.
 	 * It is important to note that you can call this function on an existing
 	 * file, but it will invalidate the file and break all subsequent uses of
 	 * the class unless you call load_file() again to reset the value back to
 	 * the one used in the file.
 	 */
     void setNY(int NY);
-	/** @param double [NX,NY,NZ] [NZ] describes the dimensions of the
+
+	/** @param double [NZ,NY,NX] [NX] describes the dimensions of the
 	 * computational grid. This function is useful when creating a new pfb file.
 	 * It is important to note that you can call this function on an existing
 	 * file, but it will invalidate the file and break all subsequent uses of
 	 * the class unless you call load_file() again to reset the value back to
 	 * the one used in the file.
 	 */
-    void setNZ(int NZ);
-
-    /**
-     * getD[X,Y,Z]
-     * [DX,DY,DZ] describe the TODO: the what?
-     * This function is useful either when reading an existing file or when confirming the configuration
-     * of a file that is being created where the computational grid has already been set.
-     * @returns int
-     */
+    void setNX(int NX);
 
-	/** param get[DX,DY,DZ] [DX] describe the TODO: the what? This function
+	/** param get[DZ,DY,DX] [DZ] describe the TODO: the what? This function
```

Comment:
It looks like there might be a leftover comment in the in the get[DZ,DY,DX] functions along with the set[DZ,DY,DZ] functions: "describe the TODO: the what?"